js/src/common/components/Page.js

```diff
@@ -1,4 +1,5 @@
 import Component from '../../common/Component';
+import PageState from '../../forum/states/PageState';
```

Comment:
It feels wrong for a component in `common` to import something from a more specific module such as `forum`. So should `PageState` be moved to `common` as well?